docs/governance/templates/release_issue.md

```diff
@@ -30,14 +30,15 @@ Complete Milestone
 ------------------
 - [ ] Create the next version milestone, use [semantic versioning](https://semver.org/) when naming it to be consistent with the [Go community](https://blog.golang.org/versioning-proposal).
 - [ ] Visit the [milestone](https://github.com/GoogleCloudPlatform/open-match/milestone).
-  - [ ] Open a document for a draft [release notes](release.md).
-  - [ ] Add the milestone tag to all PRs and issues that were merged since the last milestone. Look at the [releases page](https://github.com/GoogleCloudPlatform/open-match/releases) and look for the "X commits to master since this release" for the diff. The link resolves to, https://github.com/GoogleCloudPlatform/open-match/compare/v{version}...master.
+  - [ ] Create a *draft* release with the [release template][release-template].
+    - [ ] Make a `tag` with the release version. The tag must be v{version}. Example: v0.5.0. Append -rc# for release candidates. Example: v0.5.0-rc1.
```

Comment:
This tag should be tagging the commit for the version update, I think.  Which doesn't exist yet.

Reply:
I've changed the wording here to be easier to understand.